base/applications/network/net/main.c

```diff
@@ -45,6 +46,24 @@ COMMAND cmds[] =
 
 HMODULE hModuleNetMsg = NULL;
 
+VOID PrintTranslatedStatus(
+    NET_API_STATUS status)
+{
+    switch (status) {
+        case NO_ERROR:
+            ConResPuts(StdOut, IDS_SUCCESS);
+            break;
+        case ERROR_ALIAS_EXISTS:
+           ConResPuts(StdOut, IDS_USEREXISTS);
+           break;
+        case 2221: //FIXME: replace numeral literal
+            ConResPuts(StdOut, IDS_NOTFOUND);
+            break;
+        default:
+            ConPrintf(StdOut, L"Status: %lu\n", status);
```

Comment:
Couldn't it be possible to use `ConMsgPuts`? (see for example https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/network/netstat/netstat.c;hb=39003dd408a605b4e66f1b040c7333b52e4d8158#l55 )
EDIT: Maybe `PrintMessageString` can do the trick.

Reply:
@HBelusca Hello. I dont't know how can I put message in dll to use `PrintMessageString`.

Reply:
Just calling `PrintMessageString(status);` instead of this `ConPrintf` doesn't work?

Reply:
@HBelusca Okey, I'll try it.
I understood the aim of using this function incorrectly.